WcaOnRails/app/webpacker/components/Media/MediaTable.jsx

```diff
@@ -0,0 +1,257 @@
+import React, { useEffect, useState } from 'react';
+
+import _ from 'lodash';
+import { Table, Confirm, Modal, Button } from 'semantic-ui-react';
+import { competitionUrl, apiV0Urls } from '../../lib/requests/routes.js.erb';
+import { countries, continents, years } from '../../lib/wca-data.js.erb';
+import { dateRangeBetween } from '../../lib/helpers/media-table';
+import useLoadedData from '../../lib/hooks/useLoadedData';
+import useSaveAction from '../../lib/hooks/useSaveAction';
+import {
+  DropdownMenu,
+  DropdownItem,
+  DropdownHeader,
+  DropdownDivider,
+  Dropdown,
+} from 'semantic-ui-react'
+export default function MediaTable({ isValidate }) {
+  const countryOptions = _.map(countries.byIso2, (country) => ({
+    key: country.iso2,
+    text: country.name,
+    value: country.iso2,
+  }));
+  const continentOptions = continents.map((continent) => ({
+    key: continent.id,
+    text: continent.name,
+    value: continent.name,
+  }));
+  const yearOptions = [
+    {
+      key: 'all',
+      text: 'All Years',
+      value: 'All Years',
+    },
+    ...years.map((year) => ({
+      key: year,
+      text: year,
+      value: year,
+    })),
+  ];
```

Comment:
These could be defined outside the component (otherwise they're recomputed each re-render).

Reply:
Fixed